src/Twilio/Converters/PhoneNumberConverter.cs

```diff
@@ -17,7 +17,9 @@ public class PhoneNumberConverter : JsonConverter
         /// <param name="serializer">unsued</param>
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            var t = JToken.FromObject(value.ToString());
+            var t = value.ToString() != null
```

Comment:
value.ToString() can not be null.
Reason:
It can be only null when "value" is null, in that case value.ToString() will throw System.NullReferenceException.
What are your thoughts ?

(Exception: When we are overriding ToString() method)

Reply:
Here value is an object of PhoneNumber and has a _number variable. 
The toString() has been overriden to return _number, so it forms a case where value!=null but value.ToString() is null